src/index.js

```diff
@@ -31,76 +26,13 @@ export default class Kinto {
     };
   }
 
-  /**
-   * Synchronization strategies. Available strategies are:
-   *
-   * - `MANUAL`: Conflicts will be reported in a dedicated array.
-   * - `SERVER_WINS`: Conflicts are resolved using remote data.
-   * - `CLIENT_WINS`: Conflicts are resolved using local data.
-   *
-   * @type {Object}
-   */
-  static get syncStrategy() {
-    return Collection.strategy;
-  }
-
-  /**
-   * Constructor.
-   *
-   * Options:
-   * - `{String}`       `remote`      The server URL to use.
-   * - `{String}`       `bucket`      The collection bucket name.
-   * - `{EventEmitter}` `events`      Events handler.
-   * - `{BaseAdapter}`  `adapter`     The base DB adapter class.
-   * - `{String}`       `dbPrefix`    The DB name prefix.
-   * - `{Object}`       `headers`     The HTTP headers to use.
-   * - `{String}`       `requestMode` The HTTP CORS mode to use.
-   *
-   * @param  {Object} options The options object.
-   */
   constructor(options={}) {
     const defaults = {
       adapter: Kinto.adapters.IDB,
-      bucket: DEFAULT_BUCKET_NAME,
-      events: new EventEmitter(),
-      remote: DEFAULT_REMOTE,
+      events: new EventEmitter()
     };
-    this._options = Object.assign(defaults, options);
-    this._api = new Api(this._options.remote, {
-      headers:     this._options.headers,
-      events:      this._options.events,
-      requestMode: this._options.requestMode,
-    });
-
-    // public properties
-    /**
-     * The event emitter instance.
-     * @type {EventEmitter}
-     */
-    this.events = this._options.events;
-  }
-
-  /**
-   * Creates a Collection instance. The second (optional) parameter
-   * will set collection-level options like e.g. remoteTransformers.
-   *
-   * @param  {String} collName The collection name.
-   * @param  {Object} options May contain the following fields:
-   *                          remoteTransformers: Array of RemoteTransformers
-   * @return {Collection}
-   */
-  collection(collName, options = {}) {
-    if (!collName) {
-      throw new Error("missing collection name");
-    }
 
-    const bucket = this._options.bucket;
-    return new Collection(bucket, collName, this._api, {
-      events:              this._options.events,
-      adapter:             this._options.adapter,
-      dbPrefix:            this._options.dbPrefix,
-      idSchema:            options.idSchema,
-      remoteTransformers:  options.remoteTransformers
-    });
+    let expandedOptions = Object.assign(defaults, options);
+    super(expandedOptions);
```

Comment:
Nit: maybe directly 

```js
super(Object.assign({}, defaults, options));
```

Note that I'm also suggesting we pass it a copy instead of mutating the `default` object; not that it's impacting here, just I consider good practice to generally avoid mutations.